test/db/abi/languages/freepascal

```diff
@@ -11,6 +11,7 @@ RUN
 
 NAME=LANGUAGES: freepascal main detection
 FILE=bins/abi_bins/elf/languages/freepascal/pick_random
+BROKEN=1
```

Comment:
?

Reply:
i can confirm the main detected in rizin is wrong. there is no function at that resolved address

Reply:
It has no main, according to the IDA Pro 7.7. It has `start` which is entry point and jumps to the following function.

Reply:
Instead of marking it as BROKEN it's better to remove this test.

Reply:
Why remove the test? I think this is after all broken. If there is no main, why the flag `main` is set?